packages/unimodules-task-manager-interface/android/src/main/java/org/unimodules/interfaces/taskManager/TaskManagerInterface.java

```diff
@@ -5,6 +5,10 @@
 import java.util.Map;
 
 public interface TaskManagerInterface {
+
+  String EVENT_NAME = "TaskManager.executeTask";
+  String E_TASK_SERVICE_NOT_FOUND = "E_TASK_SERVICE_NOT_FOUND";
```

Comment:
Maybe it's a good occasion to change our error tags and prefix them with `ERR` instead of `E`? 🤔